chucks-pick3-mobile/package.json

```diff
@@ -5,19 +5,20 @@
   "homepage": "https://ionicframework.com/",
   "scripts": {
     "ng": "ng",
-    "start": "ng serve",
-    "build": "ng build",
-    "test": "ng test",
+    "serve": "npm run use-mocks && ionic serve --configuration=mock",
+    "start": "npm run use-prod && ionic serve --configuration=prod",
```

Comment:
Start script needs to call gulp task to start server; so revert this script and start again.  For example, we should have something like this: `npm run use-mock && npm run server`